src/components/Header/Tools.tsx

```diff
@@ -236,22 +231,24 @@ const Tools = () => {
     );
   };
 
+  const isNotificationsEnabled = useFlag('platform.chrome.notifications-drawer');
+  const [isNotificationDrawerExpanded, setIsNotificationsDrawerExpanded] = useAtom(notificationDrawerExpandedAtom);
+  const toggleDrawer = () => {
+    setIsNotificationsDrawerExpanded((prev) => !prev);
+  };
+
   return (
     <>
       {isNotificationsEnabled && (
-        <ToolbarItem className="pf-v6-u-mx-0">
-          <Tooltip aria="none" aria-live="polite" content={'Notifications'} flipBehavior={['bottom']} className="tooltip-inner-settings-cy">
-            <NotificationBadge
-              className="chr-c-notification-badge"
-              variant={unreadNotifications ? 'unread' : 'read'}
-              onClick={() => toggleNotifications((prev) => !prev)}
-              aria-label="Notifications"
-              isExpanded={isNotificationDrawerExpanded}
-            >
-              <BellIcon />
-            </NotificationBadge>
-          </Tooltip>
-        </ToolbarItem>
+        <ScalprumComponent
+          scope="notifications"
+          module="./NotificationsDrawerBell"
+          fallback={null}
+          {...{
```

Comment:
Is there a reason why not to use the props normally and "deconstruct" it from an object?